### ISSUE TITLE
JACK config presets for quick switch-master options

### ISSUE THREAD

**Issue by wolftune** [falkTX/Cadence#120]
I regularly change my audio hardware, and sometimes want quick combinations of certain hardware with certain latency etc. Currently, I use Claudia studios to make these changes, but then I have to turn on bridges again manually, and I also lose all my setup I might have in terms of any running programs. Going into the JACK configuration to change hardware is tedious in its own way.

Perhaps this is related to #119 but I would *love* to be able to save a configuration for Jack settings and then have all saved configurations available from a drop-down in cadence which instantly sets those settings and then does a switch-master. That would dramatically improve my whole use of KXStudio overall.

**Comment by wolftune** [comment]
Please oh please, I switch interfaces so often. I want a switch-interface option that is similar enough to switching screens for video. I could just do the quick function-key thing to swap to a different display. I want something closer to that for audio interfaces.

**Comment by naught101** [comment]
Yeah, I would find this super useful too. Mostly because on my external interface I often want a lower latency, so I can play keys/drums, but my internal soundcard can't handle it, so I need to switch back to a higher buffer size.
